src/infrastructure/storage/user.ts

```diff
@@ -50,6 +55,22 @@ export class UserStore extends SubscribableStore<UserStoreData> {
    * @param editorTools - editor plugins
    */
   public setUserEditorTools(editorTools: EditorTool[]): void {
-    this.data.editorTools = editorTools;
+    this.data.userEditorTools = editorTools;
+  }
+
+  /**
+   * Array of all tools
+   */
+  public getAllEditorTools(): EditorTool[] {
+    return this.data.allEditorTools;
+  }
+
+  /**
+   * Set all editor tools that are used in notes creation
+   *
+   * @param editorTools - editor plugins
+   */
+  public setAllEditorTools(editorTools: EditorTool[]): void {
```

Comment:
it is not related to user storage and app state. Just create the new application service, domain service, repo